### PR TITLE
Add Annotations

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: api/v1beta1/aerospikecluster_types.go [hunk 1]
```diff
@@ -136,6 +136,8 @@ func (c *AerospikeOperatorClientCertSpec) validate() error {
 
 // AerospikePodSpec contain configuration for created Aeropsike cluster pods.
 type AerospikePodSpec struct {
+	// MetaData to add to pods.
```
Comment (sud82) [inline api/v1beta1/aerospikecluster_types.go:139]: Do not use this metadata object. Create a new custom metadata struct having only annotations and labels

type ObjectMeta struct {
	Annotations 
	Labels
}

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -105,6 +105,19 @@ func LabelsForAerospikeClusterRack(clName string, rackID int) map[string]string
 	return labels
 }
 
+// MergeLabels merges operator an user defined labels
+func MergeLabels(operatorLabels, userLabels map[string]string) (map[string]string, error) {
+	for label, value := range userLabels {
+		_, ok := operatorLabels[label]
+		if ok {
+			return nil, fmt.Errorf(
```
Comment (sud82) [inline pkg/utils/utils.go:113]: Add this validation check in validationWebhook. Request should not come here if this check fails there.

File: api/v1beta1/aerospikecluster_types.go (hunk 1 above)
Comment (davi17g) [reply api/v1beta1/aerospikecluster_types.go:146]: Created new MetaObject

File: pkg/utils/utils.go (hunk 2 above)
Comment (davi17g) [reply pkg/utils/utils.go]: Done

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -95,16 +95,25 @@ func GetDesiredImage(aeroCluster *asdbv1beta1.AerospikeCluster, containerName st
...
 func LabelsForAerospikeClusterRack(clName string, rackID int) map[string]string {
 	labels := LabelsForAerospikeCluster(clName)
-	labels["aerospike.com/rack-id"] = strconv.Itoa(rackID)
+	labels[asdbv1beta1.AerospikeRackIdLabel] = strconv.Itoa(rackID)
 	return labels
 }
 
+// MergeLabels merges operator an user defined labels
+func MergeLabels(operatorLabels, userLabels map[string]string) map[string]string {
+	for label, value := range userLabels {
```
Comment (ashishshinde) [inline pkg/utils/utils.go:111]: Make a new map instead of modifying the input and returning

**Review by ashishshinde — COMMENTED** [review]
Please add a test to verify annotation and labels are applied to running pod

File: pkg/utils/utils.go (hunk 3 above)
Comment (davi17g) [reply pkg/utils/utils.go:111]: Changed

**Pull request merged**